### PR TITLE
CPO-3 : Create payment orders database 

### PR DESCRIPTION
**Pull request by sullivangareth**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CPO-3

### Change description ###

 * Chart changes to create DB in preview environment
 * Terraform scripts to create DB in all other environments 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/cpo-case-payment-orders-api/values.yaml [hunk 1]
```diff
@@ -9,3 +9,9 @@ java:
 #      secrets:
 #        - AppInsightsInstrumentationKey
   environment:
+    CPO_CASE_PAYMENT_ORDERS_DB_HOST: cpo-case-payment-orders-api-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com
+    CPO_CASE_PAYMENT_ORDERS_DB_PORT: 5432
+    CPO_CASE_PAYMENT_ORDERS_DB_NAME: cpo_case_payment_orders
+    CPO_CASE_PAYMENT_ORDERS_DB_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
+    CPO_CASE_PAYMENT_ORDERS_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
```
Comment (danlysiak) [inline charts/cpo-case-payment-orders-api/values.yaml:16]: These credentials will not work on any of the higher envs - we need to take them from the vault secrets that you're creating in the TF. 

Should be able to uncomment a few lines up in this same file where the placeholder is, and add the appropriate secret names. Then need to add them to a bootstrap.yaml, like [here](https://github.com/hmcts/aac-manage-case-assignment/blob/master/src/main/resources/bootstrap.yaml)

Comment (danlysiak) [reply charts/cpo-case-payment-orders-api/values.yaml:16]: (In fact, the first 3 are also being saved in the vault, so could really do the same for them)

Comment (sullivangareth) [reply charts/cpo-case-payment-orders-api/values.yaml]: Hopefully fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/9/commits/129f1635a9c7a21715bb7ecf0c91d34ef072b4ea

**Review by danlysiak — APPROVED** [review]

**Comment by sullivangareth** [comment]
Closing as this branch has been rebased into `https://github.com/hmcts/cpo-case-payment-orders-api/tree/CPO-3-Flyway` for PR https://github.com/hmcts/cpo-case-payment-orders-api/pull/9

**Pull request closed without merging**